### PR TITLE
Add javadocs for delete lesson related methods in LessonSchedule

### PR DESCRIPTION
**Pull request by ethantyh692**

### CODE REVIEW

File: src/main/java/tutorease/address/model/LessonSchedule.java [hunk 1]
```diff
@@ -47,21 +47,39 @@ public boolean hasLesson(Lesson lesson) {
     }
 
     /**
-     * Adds a lesson to the lesson schedule.
-     * The lesson must not already exist in the lesson schedule.
+     * Adds the specified lesson to the lesson list.
+     *
+     * @param lesson The lesson to be added. Must not be null.
+     * @throws NullPointerException if the specified lesson is null.
```
Comment (NorbertLoh) [inline src/main/java/tutorease/address/model/LessonSchedule.java:53]: not sure if needed but maybe can caps the if so that it is If

**Review by NorbertLoh — COMMENTED** [review]
Nits to fix. LGTM!

File: src/main/java/tutorease/address/model/LessonSchedule.java (hunk 1 above)
Comment (NorbertLoh) [reply src/main/java/tutorease/address/model/LessonSchedule.java:53]: yea i thought so too but our textbook suggested that
image

Comment (ethantyh692) [reply src/main/java/tutorease/address/model/LessonSchedule.java]: fixed

**Comment by NorbertLoh** [comment]
LGTM!

**Review by Chronoxy — APPROVED** [review]
LGTM! 👍

**Pull request merged**